strongbox-web-core/src/main/java/org/carlspring/strongbox/forms/PathForm.java

```diff
@@ -0,0 +1,52 @@
+package org.carlspring.strongbox.forms;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+
+public class PathForm {
+
+    @NotEmpty(message = "A storage id must be specified.")
+    @Pattern(regexp = "[a-zA-Z0-9\\-\\_\\.]+")
```

Comment:
Redundant character escapes for `_` and `.` inside square brackets.
```suggestion
    @Pattern(regexp = "[a-zA-Z0-9\\-_.]+")
```